app/src/main/java/chat/rocket/android/db/DatabaseManager.kt

```diff
@@ -401,13 +401,13 @@ class DatabaseManager(val context: Application, val serverUrl: String, val token
         }
     }
 
-    private fun mapLastMessageText(message: LastMessage?): String? = message?.let { lastMessage ->
-        if (lastMessage.message?.isEmpty() == true && lastMessage.attachments?.isNotEmpty() == true) {
-            context.getString(R.string.msg_sent_attachment)
-        } else {
-            lastMessage.message
-        }
-    }
+//    private fun mapLastMessageText(message: LastMessage?): String? = message?.let { lastMessage ->
```

Comment:
? 
Why is this required function commented?

Reply:
Same as above.